lib/models/customer_info_wrapper.dart

```diff
@@ -1,4 +1,5 @@
-import 'package:freezed_annotation/freezed_annotation.dart';
+
+import '../freezed_annotation/freezed_annotation.dart';
```

Comment:
Note I had to copy this runtime dependency in our SDK... I thought of keeping that dependency for now, but wanted to avoid incompatibilities with the freezed models down the line, considering we won't be updating those in the future. 

Plan is to slowly move away our models from freezed.